src/game/cards/crops/handlePlayFromHand.ts

```diff
@@ -0,0 +1,10 @@
+import { IGame, IPlayer } from '../../types'
+
+export const handlePlayFromHand = async (
+  game: IGame,
+  _playerId: IPlayer['id'],
+  _cardIdx: number
+) => {
+  // TODO: Implement this
```

Comment:
That's a great point. I generally have similar habits, but I considered this as a bit of an exception scenario since everything is net-new and still being figured out. That said, I think it's worth making these TODOs loud so they're not forgotten about. I've updated that for the next PR in https://github.com/jeremyckahn/farmhand-shuffle/commit/bbb36afba31dd64fe18cca4a1323610c1a4d00bb!